S21/indraneelL/refactorme.py

```diff
@@ -1,84 +1,92 @@
 import unicodedata
 
+class BaseClass:
+    CAPITAL_ACUTE_A = '\u00c1'
```

Comment:
As I mentioned in class, it would be nice to name these according the Unicode data files, e.g. here "LATIN_CAPITAL_LETTER_A_WITH_ACUTE".  Also this is a magic constant; you can embed the literal unicode character in the Python source.